doc/api/cli.md

```diff
@@ -132,6 +132,13 @@ V8 inspector integration allows tools such as Chrome DevTools and IDEs to debug
 and profile Node.js instances. The tools attach to Node.js instances via a
 tcp port and communicate using the [Chrome DevTools Protocol][].
 
+### `--loader`
+<!--
+added: v9.0.0
+-->
+
+Specify a customer loader, to load [ECMAScript Modules][].
```

Comment:
s/customer/custom

Also, it should be noted that this is still an experimental feature.

Reply:
@jdalton given you have to run node with --experimental-modules anyway to use this - why?

Reply:
@benjamingr 
> given you have to run node with --experimental-modules anyway to use this - why?

I like the consistency of the `--experimental-` prefix for experimental things. The prefix makes it explicit that the feature/sub-feature is experimental.